apps/frontend/src/pages/chats/libs/components/chat-search-bar/chat-search-bar.tsx

```diff
@@ -0,0 +1,61 @@
+import { type FormEvent } from "react";
+
+import { Input } from "~/libs/components/components.js";
+import { initDebounce } from "~/libs/helpers/helpers.js";
+import { useAppDispatch, useAppForm } from "~/libs/hooks/hooks.js";
+import {
+	type ChatSearchDto,
+	DEFAULT_SEARCH_CHAT_PAYLOAD,
+	SEARCH_CHAT_DELAY_MS,
+	actions as chatsActions,
+} from "~/modules/chats/chats.js";
+
+import styles from "./styles.module.css";
+
+const handleFormSubmit = (event_: FormEvent<HTMLFormElement>): void => {
+	event_.preventDefault();
+};
+
+const ChatSearchBar: React.FC = () => {
+	const dispatch = useAppDispatch();
+
+	const { control, errors, handleSubmit } = useAppForm({
+		defaultValues: DEFAULT_SEARCH_CHAT_PAYLOAD,
+		mode: "onChange",
+	});
+
+	const handleSearchCourses = (filterFormData: ChatSearchDto): void => {
```

Comment:
```suggestion
	const handleSearchChats = (filterFormData: ChatSearchDto): void => {
```
We are searching for chats here